### PR TITLE
Adjust getting started guide to 1.0 syntax

### PR DESCRIPTION
**Pull request by rdhananjaya**
## Purpose
$subject

- Any checklist items to be verified before production deployment?
- Special styling concerns? 
- Related media files?

### CODE REVIEW

File: website/two-column-pages/docs/learn/getting-started.md [hunk 1]
```diff
@@ -45,12 +43,6 @@ The installer should put the `C:\Program Files\Ballerina\<ballerina-directory>\b
 dpkg -i <ballerina-binary>.deb
 ```
 
-* If you downloaded the RPM file, use the following command to install Ballerina. Replace `<ballerina-home>/<ballerina-binary>.deb` with the actual file path. This installs Ballerina to the `/usr/lib64/ballerina` directory.
- 
-```
-rpm -i <ballerina-binary>.rpm
-```
-
 ## Installing via the Ballerina language ZIP file
```
Comment (anupama-pathirage) [inline website/two-column-pages/docs/learn/getting-started.md]: We need to remove this section as we no longer provide ZIP. @keizer619 Please confirm

File: website/two-column-pages/docs/learn/getting-started.md [hunk 2]
```diff
@@ -12,15 +12,13 @@ Ballerina binary distributions are available for the following supported operati
 
 - Windows Vista SP2 x64 or later
 - Ubuntu Linux 12.04 x64 - LTS and above
-- Suse Linux Enterprise Server 10 SP2 and above
-- Red Hat Enterprise Linux 5.5 x64 and above
 - OS X 10.8.3 x64 and above
 
-If your operating system or architecture is not on the list, you can [install from source](https://github.com/ballerina-platform/ballerina-lang#install-from-source) instead.
+If your operating system or architecture is not on the list, you can [install from source](https://github.com/ballerina-platform/ballerina-lang#install-from-source) or install from the zip method describe below.
```
Comment (MaryamZi) [inline website/two-column-pages/docs/learn/getting-started.md]:
```suggestion
If your operating system or architecture is not on the list, you can [install from source](https://github.com/ballerina-platform/ballerina-lang#install-from-source) or install from the zip method described below.
```

File: website/two-column-pages/docs/learn/getting-started.md [hunk 3]
```diff
@@ -12,15 +12,13 @@ Ballerina binary distributions are available for the following supported operati
...
 
 ## Installing Ballerina via installers
 
-If you are upgrading to the latest version of Ballerina from an older version, you can download the latest version and run it without uninstalling the old version. 
+If you are upgrading to the latest version of Ballerina from an older version, you can download the latest version and the installer will automatically uninstall the old version.
```
Comment (MaryamZi) [inline website/two-column-pages/docs/learn/getting-started.md]:
```suggestion
If you are upgrading to a new version of Ballerina from an older version, you can download and use the installer for the latest version. The installer will automatically uninstall the old version. 
```

File: website/two-column-pages/docs/learn/getting-started.md (hunk 1 above)
Comment (rdhananjaya) [reply website/two-column-pages/docs/learn/getting-started.md]: Got it confirmed, only the installers will be there

File: website/two-column-pages/docs/learn/getting-started.md [hunk 4]
```diff
@@ -12,15 +12,13 @@ Ballerina binary distributions are available for the following supported operati
 
 - Windows Vista SP2 x64 or later
 - Ubuntu Linux 12.04 x64 - LTS and above
-- Suse Linux Enterprise Server 10 SP2 and above
-- Red Hat Enterprise Linux 5.5 x64 and above
 - OS X 10.8.3 x64 and above
 
-If your operating system or architecture is not on the list, you can [install from source](https://github.com/ballerina-platform/ballerina-lang#install-from-source) instead.
+If your operating system or architecture is not on the list, you can [install from source](https://github.com/ballerina-platform/ballerina-lang#install-from-source) or install from the zip method described below.
```
Comment (MaryamZi) [inline website/two-column-pages/docs/learn/getting-started.md]: Need to remove zip reference here too?

```suggestion
If your operating system or architecture is not on the list, you can [install from source](https://github.com/ballerina-platform/ballerina-lang#install-from-source).
```

Comment (rdhananjaya) [reply website/two-column-pages/docs/learn/getting-started.md]: done

**Pull request merged**